service/controller/clusterapi/v18/cluster_resource_set.go

```diff
@@ -103,6 +103,29 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 	//	}
 	//}
 
+	//var certConfigResource controller.Resource
```

Comment:
Did you comment it out and not delete it deliberately?

Reply:
Yes this is on purpose. We are in the process of moving resources over to the new `clusterapi` controller to make things more provider independent in `cluster-operator`. Until we have all resources in place we do not want to run them because this would screw testing the current node pools clusters. This is a rather pragmatic approach which is also reflected in the roundhouse kick I did here in this PR. 🙈